### ISSUE TITLE
Option to generate C/C++ extension config for detected unreal project version

### ISSUE THREAD

**Issue by tilkinsc** [hcabel/UnrealVsCodeHelper#4]
We need to get intellisence set up with this plugin then its already better than msvs.

**Comment by hcabel** [comment]
Yeah that's a pretty hard problem to solve that I haven't yet figure out :/ 
I'm focusing on giving quality of life improvement tool first (project naming 🤔)
But maybe I can give an auto-setup tool of some else's intelisense
(sorry for delay, I'm working on an off on Unreal these days and by extension this plugin ^^)

**Comment by tilkinsc** [comment]
I already gave up the terrible development lifecycle of unreal for godot in that mean time :)
